docs/usage/rest_api/interpreter.md

```diff
@@ -153,8 +156,16 @@ The role of registered interpreters, settings and interpreters group are describ
       "name": "spark",
       "group": "spark",
       "properties": {
-        "spark.cores.max": "",
-        "spark.executor.memory": "1g",
+        "spark.cores.max": {
+          "name": "",
+          "value": "spark.cores.max",
+          "type": "number"
+        },
+        "spark.executor.memory": {
+          "name": "",
```

Comment:
is it ok to have name `""`? 

other settings have 

![image](https://user-images.githubusercontent.com/4968473/27779051-2d1b31b0-5ffb-11e7-800d-33770c78c931.png)